service/controller/v2/arm_templates/main.json

```diff
@@ -361,7 +410,10 @@
           "vmssOsImageSKU": {"value": "[parameters('masterNodes')[0].osImage.sku]"},
           "vmssOsImageVersion": {"value": "[parameters('masterNodes')[0].osImage.version]"},
           "vmssVmCustomData": {"value": "[parameters('masterCloudConfigData')]"},
-          "vmssLbBackendPoolId": {"value": "[reference('api_load_balancer_setup').outputs.backendPoolId.value]"},
+          "vmssLbBackendPoolIds": {"value": [
```

Comment:
Please let's change the name to `vmssLbBackendPools` - they aren't IDs any more.